.github/workflows/ci_e2e_workflow.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           AzureWebJobsEventGridTopicUri: ${{ secrets.LinuxEventGridTopicUriString37 }}
           AzureWebJobsEventGridConnectionKey: ${{ secrets.LinuxEventGridConnectionKeyString37 }}
         run: |
-          python -m pytest -n auto  --instafail --cov=./azure_functions_worker --cov-report xml --cov-branch --cov-append tests/endtoend
+          python -m pytest --reruns 4 --instafail --cov=./azure_functions_worker --cov-report xml --cov-branch --cov-append tests/endtoend
```

Comment:
Install pytest from pypi.  Change line 61 to 
python -m pip install  -U -e .[dev]

Reply:
updated as per recommendation. thanks